demos/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Demo: MyComponent</title>
+
+	<link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap.css">
+
+	<script src="../node_modules/closure-templates/soyutils.js"></script>
+	<script src="../build/globals/myComponent.js"></script>
+</head>
+<body>
+	<h1>Layout component</h1>
+	<script type="text/javascript">
+		var layout = new metal.Layout({
+		  data: [
+		    {
+		      columns: [
+		        {
+		          content: 'Column 1.1',
+		          size: 3
+		        },
+		        {
+		          content: 'Column 1.2',
+		          size: 6
+		        },
+		        {
+		          content: 'Column 1.3',
+		          size: 3
+		        }
+		      ]
+		    },
+		    {
+		      columns: [
+		        {
+		          content: 'Column 2.1',
+		          size: 8
+		        },
+		        {
+		          content: 'Column 2.2',
+		          size: 4
+		        }
+		      ]
+		    }
+		  ]
+		});
+
+		layout.render();
```

Comment:
We don't use this `render` function anymore, as you can see from [metal-modal's demo page](https://github.com/metal/metal-modal/blob/master/demos/index.html#L17)